### DISCUSSION TITLE
No NIH number, I'm Canadian.  How can i embargo my data until I publish

### DISCUSSION THREAD

**Discussion by AngCamp** [dandi/helpdesk#145] [General]
I am submitting a manuscript and the publisher requires that I have my data submitted to an online repo.  I really want to use Dandi but I was advised by my PI to embargo the data before we publish it, we're Canadian though and the embargo option seems to require submitting an NIH number, is there some alternative we can use?

**Comment by satra** [comment]
if you have a canadian project number you can put that in. if that doesn't work for the moment enter "DND 123456". also please make sure to read the terms/policies around embargo.

**Comment by AngCamp** [comment]
Hi Satra,

I accidently created a draft dataset without embargoing it, is there a way to delete it?  It's empty.  https://dandiarchive.org/dandiset/001079

Also I am trying to create a new embargoed version and DND 123456 is not working, I did not use the quotes to be clear.  I have asked my supervisor for the project number but he is away on holiday and may not reply promptly so being able to use a placeholder would be preferable to me.

![image](https://github.com/dandi/helpdesk/assets/77176433/9f81279d-138f-40f8-9bc4-f474f2015025)

**Reply by kabilar** [reply]
Hi @AngCamp, you can delete a Dandiset with the [DANDI command line interface](https://dandi.readthedocs.io/en/latest/cmdline/delete.html) or [DANDI Python API](https://dandi.readthedocs.io/en/latest/modref/generated/dandi.delete.html).

**Reply by kabilar** [reply]
Regarding the `NIH Award Number`, as a temporary workaround, please enter `1 R01 CA 123456-01` in the `NIH Award Number` field and enter the Gov of Can award number in the `Description` field.  For further details see this [reference](https://github.com/dandi/helpdesk/issues/148). Thanks.
